utils/localization/environment.yml

```
@@ -0,0 +1,7 @@
+name: mtasa
+
+dependencies:
+  - python=3.9
+  - pip
+  - pip:
+    - pytest
```

Nit: Fix missing newline at the end of the file.
```suggestion
    - pytest

```